brightwind/load/station.py

```diff
@@ -757,25 +766,25 @@ def wdir_heights(self):
         return self.get_heights(measurement_type_id='wind_direction')
 
     @staticmethod
-    def __meas_point_merge(sensor_cfgs, sensors=None, mount_arrgmts=None):
+    def __meas_point_merge(logger_measurement_cfgs, sensors=None, mount_arrgmts=None):
         """
-        Merge the properties from sensor_cfgs, sensors and mounting_arrangements. This will account for when
+        Merge the properties from logger_measurement_cfgs, sensors and mounting_arrangements. This will account for when
         each property was changed over time.
 
-        :param sensor_cfgs:   Sensor cfgs properties
-        :type sensor_cfgs:    list
-        :param sensors:       Sensor properties
-        :type sensors:        list
-        :param mount_arrgmts: Mounting arrangement properties
-        :type mount_arrgmts:  list
-        :return:              The properties merged together.
-        :rtype:               list(dict)
+        :param logger_measurement_cfgs: Sensor cfgs properties
+        :type logger_measurement_cfgs:  list
```

Comment:
We should probably use 'configs' instead of 'cfgs'.

Reply:
renamed all to logger_measurement_configs